src/node/routes/domainProxy.ts

```diff
@@ -55,7 +55,7 @@ router.all("*", (req, res, next) => {
       }
       // Redirect all other pages to the login.
       return redirect(req, res, "login", {
-        to: req.path,
+        to: req.path !== "/" ? req.path : undefined,
```

Comment:
Do we not need to worry about `baseUrl` here?

Reply:
I didn't add it since `baseUrl` is empty when the route is on the root but I think it's a good idea to add it since the route should work no matter where it's mounted.